bin/development-server.js

```diff
@@ -47,12 +43,14 @@ app.use(webpackDevMiddleware(compiler, {
   stats: { colors: true }
 }));
 
-if(feature.isEnabled("hotReloading")) {
+if (feature.isEnabled("hotReloading")) {
   app.use(webpackHotMiddleware(compiler));
+} else {
+  console.log("Hot Reloading can be enabled by adding " +
+  "\"hotReloading\": true to your local.json config");
```

Comment:
That's not a bad idea, I like it. I was trying to think of a way to encourage it without having it on the readme, and this makes sense.